hack/verify-errexit.sh

```diff
@@ -54,8 +43,8 @@ tmp=$(mktemp)
 rm $tmp
 
 for file in ${shFiles}; do
-  grep "set -o errexit" $file > /dev/null 2>&1 && continue
-  grep "set -[a-z]*e" $file > /dev/null 2>&1 && continue
+  grep "^set -o errexit" $file > /dev/null 2>&1 && continue
+  grep "^set -[a-z]*e" $file > /dev/null 2>&1 && continue
```

Comment:
note: this change allows the script to run correctly on itself, since previously it would've seen the `set -o errexit` in this grep command!